girder/models/upload.py

```diff
@@ -80,11 +81,17 @@ def uploadFromFile(self, obj, size, name, parentType=None, parent=None,
         :param assetstore: An optional assetstore to use to store the file.  If
             unspecified, the current assetstore is used.
         :type reference: str
+        :param attachParent: if True, instead of giving the file a parentType
```

Comment:
Strictly speaking, files don't have a `parent` or `parentType` property. Rather, the `itemId` property of the file is getting set to `None`.